pkgs/applications/misc/freeplane/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, jdk, jre, gradle, git }:
+
+stdenv.mkDerivation rec {
+  name = "freeplane-1.6.13";
+
+  src = fetchFromGitHub {
+    owner = "freeplane";
+    repo = "freeplane";
+    rev = "b14e1dfa360453209807ebc6944fb6f3828fb4a3";
+    sha256 = "0jmmw185bv3cs036x9gj5zn1kgggfcphn7b286hw1nrnpjnxj2cq";
+  };
+
+  buildInputs = [ jdk gradle git ];
+
+  buildPhase = ''
+    export GRADLE_USER_HOME="/tmp"
+    gradle dist
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{bin,nix-support}
+    cp -r BIN $out/nix-support
+    sed -i 's/which/type -p/' $out/nix-support/BIN/freeplane.sh
+
+    cat >$out/bin/freeplane <<EOF
```

Comment:
consider using `makeWrapper` here.

Reply:
@Mic92 I found https://nixos.org/nixpkgs/manual/#ssec-stdenv-functions but I do not understand how to use it in this case.

Reply:
@puffnfresh explained to me how it works. I'll update as soon as Freeplane .desktop PR is done.

Reply:
@Mic92 Updated to use `makeWrapper` and also to use .desktop file